### DISCUSSION TITLE
Missing Burndown charts + extras

### DISCUSSION THREAD

**Discussion by luisamd** [community/community#11981] [Projects and Issues]
### Minor Improvements for Github Projects

* It should be able to have the option to generate burndown charts for optimal SCRUM adoption.
* We, as users should have the possibility to monitor the flow of the ongoing sprint and consult the past Sprints

**Comment by conatus** [comment]
Does anyone know how one would set this up in the now released Charts feature?

I want:
- Single cycle
- By days across cycle
- Amount of issues in a particular status

Can't quite see how to do this...

**Comment by andreaslech** [comment]
Four years later and GitHub Projects still doesn't have native burndown charts or sprint flow monitoring... I have been trying lots of things and I am excited about how far you can get with GitHub Copilot out of the box!

I put together a small open-source repo with ready-to-use prompts:
[github.com/Flow2CHQ/copilot-agile-metrics](https://github.com/Flow2CHQ/copilot-agile-metrics)

Burndown chart, sprint analysis, past sprint retro input, stakeholder update: works directly in VS Code or the GitHub web UI, no extra tools or API setup needed.
